src/openrct2/world/footpath.c

```diff
@@ -346,7 +346,7 @@ static money32 footpath_place_real(sint32 type, sint32 x, sint32 y, sint32 z, si
         return MONEY32_UNDEFINED;
     }
 
-    if (flags & GAME_COMMAND_FLAG_APPLY)
+    if ((flags & GAME_COMMAND_FLAG_APPLY) && (flags & GAME_COMMAND_FLAG_7) && !(flags & GAME_COMMAND_FLAG_GHOST))
```

Comment:
When the ghost piece was active and I placed it somewhere it started to constantly spam the command without the ghost flag, so I made it only invalidates when the flag is present. I will take a look into it.